client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -78,9 +94,12 @@ export const receiveSuccess = ( { dispatch }, { siteId, postId }, revisions ) =>
 	const normalizedRevisions = map( revisions, normalizeRevision );
 
 	forEach( normalizedRevisions, ( revision, index ) => {
-		revision.changes = countDiffWords(
-			diffWords( get( normalizedRevisions, [ index + 1, 'content' ], '' ), revision.content )
-		);
+		const previousRevision = get( normalizedRevisions, index + 1, {} );
+		revision.changes = {
```

Comment:
I like these changes, good work :)

Reply:
(That's not even a pun)

Reply:
Also it looks like I'll be able to use `dispatch( withAnalaytics( ... ) )` after this to track the diff generation time :)